eng/mgmt.proj

```diff
@@ -1,4 +1,7 @@
 <Project DefaultTargets="Build" InitialTargets="VerifyScope" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
I'm not sure what the difference between this file and mgmt.proj is really for but someone was using this directly so I'm updating it the same way as the root mgmt.proj. It would be nice to consider removing one or the other as well as evaluate whether or not build.proj in the root is useful.